### PR TITLE
Added Surfshark documentation for authentication

### PR DESCRIPTION
**Pull request by WoLpH**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,7 @@
 ### Surfshark
 
 - Pick the [region](https://github.com/qdm12/private-internet-access-docker/wiki/Surfshark) or a multi hop region name
+- Fetch the `username` and `password` from the bottom of the [manual setup](https://account.surfshark.com/setup/manual) page.
```
Comment (qdm12) [inline README.md:66]:
```suggestion
```

I would keep that information in the environment variables table where it's useful (as you did as well 👍 )

File: README.md [hunk 2]
```diff
@@ -179,8 +180,8 @@ Want more testing? ▶ [see the Wiki](https://github.com/qdm12/private-internet-
 
     | Variable | Default | Choices | Description |
     | --- | --- | --- | --- |
-    | 🏁 `USER` | | | Your username |
-    | 🏁 `PASSWORD` | | | Your password |
+    | 🏁 `USER` | | | Your **service** username. Can be found at the bottom of the [manual setup](https://account.surfshark.com/setup/manual) page |
```
Comment (qdm12) [inline README.md:183]:
```suggestion
    | 🏁 `USER` | | | Your **service** username, found at the bottom of the [manual setup page](https://account.surfshark.com/setup/manual) |
```

Just being a short line maniac, forgive me 😄

Comment (WoLpH) [reply README.md]: Normally my editor would have done that for me. The github editor is rather limited though ;)

File: README.md (hunk 1 above)
Comment (WoLpH) [reply README.md:66]: As you wish :) My thought was that having it in multiple places increases the odds of people finding it (and decreases the amount of created issues)

File: README.md (hunk 2 above)
Comment (qdm12) [reply README.md]: Your editor simplifies sentences?! I'm listening

File: README.md (hunk 1 above)
Comment (qdm12) [reply README.md]: Agreed, although that's a list of features (if I recall correctly) and then I'd have to add it for other providers.

Plus, fun fact, the Docker Hub API limits readmes to 25KB which I've reached with this repository so it would no longer sync the readme :laughing:  So I'm saving bytes now haha

File: README.md (hunk 2 above)
Comment (WoLpH) [reply README.md]: Let me rephrase, normally I would have noticed the long lines and would have tried to simplify ;) I have a big red line at 80 characters. Although when it comes to style guides, pre-commit might be useful: https://pre-commit.com/

Comment (qdm12) [reply README.md]: Ah ok that makes sense! I used some linters for my code telling me to make lines shorter but it's really a case to case. Especially in the readme, the line length doesn't really matter. It's more of a human phrase cognitive complexity let's say. And that silly 25KB limit :smile:
